### ISSUE TITLE
RTD builds are consistently timing out

### ISSUE THREAD

**Issue by larrybradley** [astropy/photutils#699]
It's typically taking between 600 and 800 s just to create the environment (i.e. before the `build_docs` command is even started).  Unfortunately, the RTD timeout is 900 s.

Note that many years back RTD granted `astropy` an extended timeout of 1500 s, and I see that recently those RTD builds are also failing rather frequently due to timeouts.

Nothing I've tried has fixed the issue.  Does anyone have any ideas?

For example, can `astropy`/Numfocus pay for better RTD service for our projects?

CC: @bsipocz, @saimn

**Comment by saimn** [comment]
That's one of the downside of having conda reinstalling so many system libraries, and using conda-forge, creating a new environment is slow. I'm giving a try with pip to see if it is better. Also it seems that the doc build use several external resources and is also quite long (it took 5 minutes on my laptop).

**Comment by bsipocz** [comment]
I've directly added in master `nomlk` to the RTD dependencies, it may have some effects of fastening up the download/build.

**Comment by bsipocz** [comment]
Actually. Have a look at the bottom of the latest run, is that a red herring?
https://readthedocs.org/api/v2/build/7580181.txt

```
[rtd-command-info] start-time: 2018-08-02T13:06:40.787189, end-time: 2018-08-02T13:07:12.868015, duration: 32, exit-code: 137
python /home/docs/checkouts/readthedocs.org/user_builds/photutils/conda/latest/bin/sphinx-build -T -E -b readthedocs -d _build/doctrees-readthedocs -D language=en . _build/html
Command killed due to excessive memory consumption
```

**Comment by saimn** [comment]
@bsipocz - You mean about the memory consumption ? I thought the same but @larrybradley is right, the issue is that the build takes too long (on the top of the build pages there is a message 'the build took 900 seconds' or so).

**Comment by larrybradley** [comment]
The emails I get from RTD say "Error: Build exited due to time out" and the build page using gives build times of ~903 s, so I assumed that timeouts were the issue.

**Comment by saimn** [comment]
I could build with pip :  :slightly_smiling_face: 
( 831 sec so not a lot of margin, but following builds should use pip's cache I guess)
https://readthedocs.org/projects/photutils-test/builds/7580522/
https://photutils-test.readthedocs.io/en/latest/epsf.html

**Comment by bsipocz** [comment]
wow @saimn, that's better than what we have now.

**Comment by larrybradley** [comment]
Thanks, @saimn!  Want to open an PR changing RTD from conda to pip?

BTW, I noticed an issue in the ePSF docs from your page, fixed in #700.  :)

**Comment by astrofrog** [comment]
The main part of the docs still take 525s to build which seems excessive.
